pom.xml

```diff
@@ -135,9 +135,73 @@
             <artifactId>spotbugs-annotations</artifactId>
             <version>4.0.1</version>
         </dependency>
+<!--        <dependency>-->
+<!--            <groupId>com.fasterxml.jackson.core</groupId>-->
+<!--            <artifactId>jackson-databind</artifactId>-->
+<!--            <version>2.9.10.1</version>-->
+<!--            <scope>compile</scope>-->
+<!--        </dependency>-->
+        <dependency>
+            <groupId>org.thymeleaf</groupId>
+            <artifactId>thymeleaf</artifactId>
+            <version>3.0.6.RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.jknack</groupId>
+            <artifactId>handlebars</artifactId>
+            <version>4.0.6</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.sf.jtidy</groupId>
+            <artifactId>jtidy</artifactId>
+            <version>r938</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.xhtmlrenderer</groupId>
+            <artifactId>flying-saucer-core</artifactId>
+            <version>9.1.6</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.xhtmlrenderer</groupId>
+            <artifactId>flying-saucer-pdf-itext5</artifactId>
```

Comment:
if possible, we should use the itext2 module. itext5 has licensing issues, so unless we need it, i'd avoid it.

Reply:
thanks for bringing this up David! I saw that itext2 had some unfixed security bugs and is no longer maintained and flying saucer, updated 2 years ago,(https://github.com/flyingsaucerproject/flyingsaucer) recommended avoiding it. Also this thread went into details a bit more: https://github.com/flyingsaucerproject/flyingsaucer/pull/123.
The bug is related to xml external entity attacks: https://seclists.org/bugtraq/2017/Nov/20. 

It doesn't seem like it would matter in our use case since we're not parsing pdfs (https://github.com/javamelody/javamelody/issues/780) but I decided to just use flying-saucer-openpdf based on recommendations, it meets our needs for now, and seems well maintained.